CRM/Core/BAO/ConfigSetting.php

```diff
@@ -118,7 +116,9 @@ public static function retrieve(&$defaults) {
         }
       }
 
-      CRM_Core_BAO_ConfigSetting::applyLocale(Civi::settings($domain->id), $domain->locales);
+      if (!$isUpgrade) {
+        CRM_Core_BAO_ConfigSetting::applyLocale(Civi::settings($domain->id), $domain->locales);
```

Comment:
Gut reaction: there are two problems with this:
  * The preceding selectAdd() does not specify `locales`m so `$domain->locales` will be undefined.
  * This is a nested `if()` statement (outer conditional `if ($domain->config_backend)` and inner conditional `if (!$isUpgrade)`). But the two conditions are contradictory -- because we only select `config_backend` in a case where `$isUpgrade`.